src/main/java/com/cloudbees/dockerpublish/DockerBuilder.java

```diff
@@ -292,9 +303,9 @@ public String getPassword() {
         public String getRegistryUrl() { return registryUrl; }
```

Comment:
All these getters can be deleted, unless you plan to do something with `readResolve` (in which case I would anyway suggest wrapping these fields in a single call returning a preconfigured `DockerRegistryEndpoint`).